src/overload_connectivity.jl

```diff
@@ -1,20 +1,23 @@
 ## 1-to-1
 
-function connectivity_tracer_1_to_1(
+@noinline function connectivity_tracer_1_to_1(
     t::T, is_influence_zero::Bool
 ) where {T<:ConnectivityTracer}
-    s = inputs(t)
-    s_out = connectivity_tracer_1_to_1(s, is_influence_zero)
-    return T(s_out)
+    if isemptytracer(t) # TODO: add test
+        return t
+    else
+        i_out = connectivity_tracer_1_to_1_inner(inputs(t), is_influence_zero)
+        return T(i_out) # return tracer
+    end
 end
 
-function connectivity_tracer_1_to_1(
+function connectivity_tracer_1_to_1_inner(
```

Comment:
that's not an additional layer, just a renaming, right?

Reply:
This might be a bit confusing to review:
* `connectivity_tracer_1_to_1_inner` is the original layer, previously called `connectivity_tracer_1_to_1`
* `connectivity_tracer_1_to_1` is a new layer that only handles the `isempty` field